example/spec/setup/PopulateProvidersCollections.js

```diff
@@ -2,9 +2,7 @@
 
 const path = require('path');
 const { addProviders, addCollections, addRules } = require('@cumulus/integration-tests');
-const { s3 } = require('@cumulus/common/aws');
-const { loadConfig } = require('../helpers/testUtils');
-const fs = require('fs-extra');
+const { loadConfig, uploadTestDataToBucket } = require('../helpers/testUtils');
```

Comment:
Does this file have any references to `uploadTestDataToBucket`?

Reply:
No, it used to. Will remove.